doc/content/getting-started/migrating-from-v2/major-changes.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Major Changes in The Things Stack
+weight: 10
+---
+
+Before getting started, we will discuss major breaking changes between {{% ttnv2 %}} and {{% tts %}}, along with some guidelines to make the migration process easier to manage.
+
+### Architecture
+
+Behind the scenes, {{% tts %}} is using a new architecture. See {{< ref "/reference/components" >}} for a complete overview of the different components of {{% tts %}}.
+
+### LoRaWAN support
+
+{{% tts %}} requires the LoraWAN MAC Protocol Version and Regional Parameters(LoraWAN PHY version) to be set per device. These default to `MAC_V1_0_2` and
+`MAC_PHY_V1_0_2_REV_B` for devices imported from {{% ttnv2 %}}.
+
+The LoraWAN MAC settings are configurable per end device. This means that all end devices need a frequency plan. Above, we used the `EU_863_870` frequency plan, but this needs to be replaced with the frequency plan corresponding to your region. A list of supported frequency plan IDs in [the lorawan-frequency-plans Github repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/blob/master/frequency-plans.yml).
+
+Furthermore, {{% tts %}} brings full support for all LoraWAN versions (1.0.0 through 1.1.0), as well as Class B and Class C devices.
```

Comment:
Casing

I would remove the version range as it's subject to change and sometimes versions get inserted (like 1.0.4 while there was 1.0.3 and 1.1 already)

Reply:
Added list of versions instead. Maybe we can drop this too, and just say "All LoRaWAN versions"?

Reply:
I prefer the latter indeed

Reply:
Updated accordingly